maja_newsletter/management/commands/send_newsletter_all_messages_continuous.py

```diff
@@ -0,0 +1,31 @@
+"""Command for sending the newsletter"""
```

Comment:
I don't think we need this command.
We can simply change the existing ``send_newsletter_continuous`` to use ``send_all=True``